### PR TITLE
Tt 1491 sort alphabetically with equal scores

### PR DESCRIPTION
**Pull request by gauravdott**

### CODE REVIEW

File: company/search.py [hunk 1]
```diff
@@ -71,7 +71,10 @@ class CompanyDocument(Document):
     logo = field.Keyword(index=False, store=True)
     has_single_sector = field.Boolean()
     modified = field.Date(index=False)
-    name = field.Text(copy_to='wildcard')
+
+    ordering_name = field.Keyword()
+    name = field.Text(copy_to=['wildcard', 'ordering_name'], boost=2)
+
```
Comment (richtier) [inline company/search.py]: please remove extra new lines.

File: company/views.py [hunk 2]
```diff
@@ -323,6 +323,18 @@ def get(self, request, *args, **kwargs):
             .search()
             .filter('term', is_published_investment_support_directory=True)
             .query(query)
+            .sort(
+                {
+                    "_score": {
+                        "order": "desc"
+                    }
```
Comment (richtier) [inline company/views.py]: this is more readable:
```
        .sort({
            {"_score": {"order": "desc"}},
            {"ordering_name": {"order": "asc"}
         })
```

File: company/search.py (hunk 1 above)
Comment (gauravdott) [reply company/search.py]: Done!

File: company/views.py (hunk 2 above)
Comment (gauravdott) [reply company/views.py]: Done.

File: company/helpers.py [hunk 3]
```diff
@@ -187,9 +187,27 @@ def build_search_company_query(params):
...
+                'bool',
+                must=must,
+                should=should,
+                minimum_should_match=1 if should else 0
+            ),
+            functions=[
+                SF({
+                    'weight': 5,
+                    'filter': Q('match', name=term)
+                })
```
Comment (richtier) [inline company/helpers.py]: can you please also add `match_phrase` too in order to support partial matches across words. something like
```
                SF({
                    'weight': 5,
                    'filter': Q(
                        'bool',
                        should=[
                            Q('match_phrase', name=term)),
                            Q('match', name=term)),
                       ]
                   }),
```

Comment (gauravdott) [reply company/helpers.py]: Done!

**Pull request merged**